Installing-and-Configuring/BootOptions.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Configuring Boot Options
+permalink: /BootOptions/
+parent: Installing and configuring
+nav_order: 85
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+
+Boot config files
+===
+
+A user has the option to make persistent modifications to the non-Qubes boot
+ process by creating one or more of the following files:
+
+| file | description |
+| ---- | ---- |
```

Comment:
@jtmoree-github-com I think this is the source of the table being screwed up. Can you please put on same line instead of being on two lines?

Reply:
@tlaurion Those dashed rows don't show up when I render the page.  I took them out.  I don't know what it is that you are seeing that you don't like.  If you want the table to render differently I suggest that you do a separate PR and I can pull from there.